### PR TITLE
DEV-5270 alerts api specification

### PR DESCRIPTION
**Pull request by nasis**

### CODE REVIEW

File: alerts/README.md [hunk 1]
```diff
@@ -0,0 +1,393 @@
...
+```
+$ curl -XPOST 'https://api.logz.io/v1/alerts'  
+  --header "X-API-TOKEN : your-api-access-token" 
+  --header "Content-Type: application/json"
+  -d '{
+        "severity": "LOW",
+        "title": "Error level logs",
+        "description": "Capture ERROR level logs in the given time range",
+        "query_string": "loglevel:ERROR",
+        "filter": "{\"bool\":{\"must\":[{\"range\":{\"@timestamp\":{\"gte\":1472454810934,\"lte\":1472458410934}}}],\"must_not\":[]}}",
```
Comment (segalziv) [inline alerts/README.md]: for simplicity I suggest to remove the timeframe filter - the reason is that this is an edge use case, which will make the reader wonder whether they must provide a timestamp range filter. maybe just add a simple match filter for type:mylogtype

File: alerts/README.md [hunk 2]
```diff
@@ -0,0 +1,393 @@
...
+    "valueAggregationField": null,
+    "groupByAggregationFields": [],
+    "alertNotificationEndpoints": [
+        0
+    ]
+}
+```
+
+### Request Header:
+- X-API-TOKEN : contains a token in order to access the API, which can be generated here - https://app.logz.io/#/dashboard/account/tokens
```
Comment (segalziv) [inline alerts/README.md:74]: can we update the search API as well to rename X-USER-TOKEN to X-API-TOKEN ?

File: alerts/README.md [hunk 3]
```diff
@@ -0,0 +1,393 @@
...
+						"format": "date-time"
+					}
+				}
+			}
+		}
+	}
+}
+```
+
+### Errors
```
Comment (segalziv) [inline alerts/README.md:385]: maybe we can be a bit clearer on the errors. not a must for now

**Review by segalziv — APPROVED** [review]
looks good

File: alerts/README.md (hunk 2 above)
Comment (nasis) [reply alerts/README.md:74]: search API is on the old rest framework

**Pull request merged**